schema/daqconf/confgen.jsonnet

```diff
@@ -31,42 +32,49 @@ local cs = {
-    s.field( "op_env", self.string, default='swtest', doc="Operational environment - used for raw data filename prefix and HDF5 Attribute inside the files"),
-    s.field( "data_request_timeout_ms", self.count, default=1000, doc="The baseline data request timeout that will be used by modules in the Readout and Trigger subsystems (i.e. any module that produces data fragments). Downstream timeouts, such as the trigger-record-building timeout, are derived from this."),
+    s.field( "process_manager", self.pm_choice, default="ssh", doc="Choice of process manager"),
+
+    # K8S
+    s.field( "k8s_image", self.string, default="dunedaq/c8-minimal", doc="Which docker image to use"),
+    s.field( "k8s_rte", self.rte_choice, default="auto", doc="0 - Use an RTE script if not in a dev environment, 1 - Always use RTE, 2 - never use RTE"),
+
+    # Connectivity Service
     s.field( "use_connectivity_service", self.flag, default=true, doc="Whether to use the ConnectivityService to manage connections"),
     s.field( "start_connectivity_service", self.flag, default=true, doc="Whether to use the ConnectivityService to manage connections"),
     s.field( "connectivity_service_threads", self.count, default=2, doc="Number of threads for the gunicorn server that serves connection info"),
     s.field( "connectivity_service_host", self.host, default='localhost', doc="Hostname for the ConnectivityService"),
     s.field( "connectivity_service_port", self.port, default=15000, doc="Port for the ConnectivityService"),
-    s.field( "connectivity_service_interval", self.count, default=1000, doc="Publish interval for the ConnectivityService"),
-    s.field( "RTE_script_settings", self.three_choice, default=0, doc="0 - Use an RTE script iff not in a dev environment, 1 - Always use RTE, 2 - never use RTE"),
+    s.field( "connectivity_service_interval", self.count, default=1000, doc="Publish interval for the ConnectivityService")
+    ]),
+
+
+  daq_common :  s.record("daq_common", [
+    s.field( "data_request_timeout_ms", self.count, default=1000, doc="The baseline data request timeout that will be used by modules in the Readout and Trigger subsystems (i.e. any module that produces data fragments). Downstream timeouts, such as the trigger-record-building timeout, are derived from this."),
     s.field( "use_data_network", self.flag, default = false, doc="Whether to use the data network (Won't work with k8s)"),
-  ]),
+    s.field( "data_rate_slowdown_factor",self.count, default=1, doc="Factor by which to suppress data generation. Former -s"),
+  ], doc="Cmmon daq_common settings"),
+
+  detector :  s.record("detector", [
+    s.field( "op_env", self.string, default='swtest', doc="Operational environment - used for HDF5 Attribute inside the files"),
+    s.field( "clock_speed_hz", self.freq, default=62500000),
+    s.field( "tpg_channel_map", self.tpg_channel_map, default="PD2HDChannelMap", doc="Channel map for TPG"),
```

Comment:
You have changed the default channel map, right?

Reply:
I have indeed. It used to be

```s.field( "tpg_channel_map", self.tpg_channel_map, default="ProtoDUNESP1ChannelMap", doc="Channel map for TPG"),```

on the trigger configuration record.

Considering that there are no more ProtoWIBs around, `PD2HDChannelMap`  seemed more appropriate...

Reply:
Hm, this made me realise I should have changed the name to `tpc_channel_map`, damn.